ComposureWatch/src/Components/Body.jsx

```diff
@@ -1,7 +1,5 @@
 const Body = (props) => {
```

Comment:
Something to consider is using destructuring instead of a props object. I like it because it makes it easy to see what props your component has/requires and it makes things easier to read when injecting them elsewhere in your component.

For example:

```
const Body = ({ className }) => {
  const classes = `whatever {className}`;
}
```

Reply:
I went through and deconstructed all the props I could find.  There is one instance where I need the whole props object that I left in place.  Also, is there a way to deconstruct an onHandler being passed in as a prop?